src/express.ts

```diff
@@ -0,0 +1,16 @@
+import * as express from "express";
+import { IResponse } from "./responses";
+
+/**
+ * Convenience method that transforms a function (handler),
+ * which takes an express.Request as input and returns an IResponse,
+ * into an express controller.
+ */
+export function toExpressHandler<T>(
+  handler: (req: express.Request) => Promise<IResponse<T>>
+): <P>(req: express.Request, res: express.Response, object?: P) => void {
+  return async (req, res, object) => {
+    const response = await handler.call(object, req);
```

Comment:
I don't think binding the `handler` to an object should be a concern of this function - if needed, it should be a responsibility of the logic that provides the `handler` to `toExpressHandler`

Reply:
this is intended as drop in replacement to the [referred method](https://github.com/teamdigitale/italia-backend/blob/master/src/app.ts#L70) already used in the italia-backend project. if we drop the object parameter we must do a refactor there cc @lussoluca

Reply:
yes let's do that